posthog/temporal/data_imports/external_data_job.py

```diff
@@ -197,17 +214,13 @@ async def run(self, inputs: ExternalDataWorkflowInputs):
             # Create source templates
             await workflow.execute_activity(
                 create_source_templates,
-                CreateSourceTemplateInputs(team_id=inputs.team_id, run_id=run_id),
+                CreateSourceTemplateInputs(team_id=inputs.team_id, run_id=job_id),
                 start_to_close_timeout=dt.timedelta(minutes=10),
                 retry_policy=RetryPolicy(maximum_attempts=2),
             )
 
         except exceptions.ActivityError as e:
-            if isinstance(e.cause, exceptions.CancelledError):
-                update_inputs.status = ExternalDataJob.Status.CANCELLED
```

Comment:
Why did we remove this check? The activity could still be cancelled for non-billing limit reasons, wouldn't we still want to set the status of the `ExternalDataJob` to `CANCELLED`?

Reply:
If the workflow gets cancelled for whatever reason, then the status will now get set to `failed` instead of cancelled:
1. Cancelled has no real meaning to users - it's effectively the same as "the job failed, go check the logs and retry it"
2. We're repurposing cancelled to mean "billing limits"